src/Apps/__stories__/Apps.story.tsx

```diff
@@ -14,11 +15,23 @@ storiesOf("Apps", module)
     )
   })
   .add("Artist Page", () => {
+    const filterState = new FilterState()
+    // TODO: This currently only changes the UI, the first fetch isn’t taking
+    //       this state into account.
+    filterState.state.for_sale = true
     return (
       <StorybooksRouter
         routes={artistRoutes}
         initialRoute="/artist2/andy-warhol"
-        initialState={{ mediator: { trigger: x => x } }}
+        initialAppState={{ mediator: { trigger: x => x } }}
+        initialState={[filterState]}
+        subscribe={{
+          to: [FilterState],
+          onChange: x => {
+            // tslint:disable-next-line:no-console
+            console.log(x.state)
+          },
```

Comment:
This currently also triggers on the first render. Not sure if we want that, we’ll have to see if the user (host app) ends up ignoring it every time.